pkg/kubelet/kuberuntime/kuberuntime_sandbox_linux_test.go

```diff
@@ -33,6 +33,7 @@ import (
 func TestApplySandboxResources(t *testing.T) {
 	_, _, m, err := createTestRuntimeManager()
 	m.cpuCFSQuota = true
+	setCgroupVersionDuringTest(cgroupV1)
```

Comment:
is there differences between v1 and v2 for these tests?

Reply:
The only difference is that in v2 the Unified map contains `"memory.oom.group": "1"`.

I've now extended these tests to check both v1 and v2 versions :+1: